README.md

```diff
@@ -288,19 +288,42 @@ use Yiisoft\Yii\Middleware\Locale;
 $middleware = $middleware->withSessionName('_custom_name');
 ```
 
-or additionally save it to cookies:
+or additionally, save it to cookies:
 
 ```php
 use Yiisoft\Yii\Middleware\Locale;
 
 /** @var Locale $middleware */
 $middleware = $middleware    
     ->withCookieDuration(new DateInterval('P30D')) // Key parameter for activating saving to cookies.
-    // Additional customization.
+    // Extra customization.
     ->withCookieName('_custom_name')
     ->withSecureCookie(true)
 ```
 
+To configure more services, such as translator, use `LocaleEvent`:
+
+```php
+use Yiisoft\EventDispatcher\Provider\Provider;
+use Yiisoft\Yii\Middleware\Event\LocaleEvent;
+
+final class TranslatorLocaleEventHandler
+{
+    public function __construct(
+        Provider $provider,
+        private Translator $translator
+    )
```

Comment:
It would be good to mention that translator requires additional package and add the link to it.